web/backend/src/Server.ts

```diff
@@ -175,78 +176,98 @@ app.get('/api/personal_info', (req, res) => {
   }
 });
 
-function isAuthorized(roles: string[], req: express.Request): boolean {
-  if (!req.session || !req.session.userid) {
-    return false;
-  }
+// function isAuthorized(roles: string[], req: express.Request): boolean {
+// return true;
+// if (!req.session || !req.session.userid) {
+// return false;
+// }
 
-  const { role } = req.session;
+// const { role } = req.session;
 
-  return roles.includes(role as string);
-}
+// return roles.includes(role as string);
+// }
```

Comment:
feel free to delete code that is not relevant anymore. Commented out code is generally frowned upon 😄